tests/ui-internal/collapsible_span_lint_calls.fixed

```diff
@@ -1,4 +1,4 @@
-#![deny(clippy::internal)]
+#![deny(clippy::collapsible_span_lint_calls)]
```

Comment:
Just because I'm curious, were changes like this necessitated by the move? Or is this just to clarify the `deny`'s purpose?

Reply:
The internal category no longer exists.